bridge-sdk/connectors/omni-connector/src/omni_connector.rs

```diff
@@ -62,6 +58,37 @@ impl OmniConnector {
         Self::default()
     }
 
+    async fn extract_transfer_log(
```

Comment:
I'm not sure if there are specific guidelines but I tend to put public functions first in the file. This way it's easier to see the module's API

Reply:
Oh, I meant the opposite to move it down as it's private. I don't mind it being part of the structure